sdk/servicebus/Directory.Build.props

```diff
@@ -28,7 +28,7 @@
     Azure messaging client libraries.
   -->
   <PropertyGroup Condition="'$(SignAssembly)' == 'true'">
-    <AssemblyOriginatorKeyFile>$(MSBuildThisFileDirectory)assets\azure-messaging.snk</AssemblyOriginatorKeyFile>
```

Comment:
This is intended to be local to the client library.   I'm not sure that it's a good idea to leave it at a level where it would impact the management areas.

Reply:
The management Library has not yet been restructured. For now it remains as it was.

Reply:
Yes we should probably scope it do just the one library project that uses it.

Reply:
Looking we should probably just move this file or all the settings into the Microsoft.Azure.ServiceBus project.

Reply:
In the existing structure, it lives at the root  container for the client library (in `data-plane`) so that it can be applied to each of the projects under that solution.   For Service Bus, that's just two project-  the library and its tests.   For other areas, the client library is broken into several projects + the associated test projects.

Reply:
OK so it should be correctly scoped to what needs it. If there is sharing between the src and tests then putting it in the Microsoft.Azure.ServiceBus directory is the correct thing to do.